<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>407</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>407</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table_device">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>DeviceName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SwipeNumber</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
